packages/ag-charts-community/src/chart/series/polar/pieSeries.ts

```diff
@@ -413,6 +414,16 @@ export class PieSeries extends PolarSeries<PieNodeDatum, Sector> {
             ],
         });
 
+        // AG-9879 Warning about missing data.
+        for (const valueDef of this.processedData?.defs?.values ?? []) {
+            // The 'angleRaw' is an undocumented property for the internal implementation, so ignore this.
+            // If any 'angleRaw' values are missing, then we'll also be missing 'angleValue' values and
+            // will log a warning anyway.
+            if (valueDef.id !== 'angleRaw' && valueDef.missing !== undefined && valueDef.missing > 0) {
+                Logger.warn(`Missing ${valueDef.missing} ${String(valueDef.property)} value(s)`);
```

Comment:
Nit: Use `Logger.warnOnce()` to avoid spamming the console.

Reply:
Done.